input/OFCB/test1/pb1.json

```diff
@@ -0,0 +1,93 @@
+{
+	"materials": {
+		"pwr_uox": {"attributes": {"recipe": true}, 
+			"constraints": [ 
```

Comment:
Yes. Please remove the tab key from your keyboard whenever you open a text editor. Kindly replace it when you open a shell :)